src/main/resources/swagger/rawls.yaml

```diff
@@ -1088,6 +1088,28 @@ paths:
             - openid
             - email
             - profile
+  '/api/workflow_queue_status':
```

Comment:
We should use a consistent style for URLs. I filed [GAWB-673](https://broadinstitute.atlassian.net/browse/GAWB-673) for this, but I wouldn't complain if you changed this to `workflow-queue-status` in the meantime.